scripts/postinstall.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const commitlintrcPath = `${process.env.INIT_CWD}/.commitlintrc.js`;
+
+fs.open(commitlintrcPath, 'wx', err => {
+  if (err) {
+    if (err.code === 'EEXIST') {
+      // silent error, don't do anything the file already exist
+      return;
+    }
+
+    throw err;
+  }
+
+  fs.writeFile(
+    commitlintrcPath,
+    "module.exports = require('@weahead/commitlint-config');",
+    err => {
+      if (err) {
+        throw err;
+      }
+      console.log('commitlintrc.js was created');

```

Comment:
@linusklingzell I'm not sure of the log output, I like to be informative but also I'm quite sure it will just disappear in the "installation log noise". Having said that leave it in but change to the resolved path of the file: `path.resolve(commitlintrcPath)`

Reply:
I removed it, was added when I tested and left it because it didn't matter if it was there or not for me and as you sad it gets lost in the "installation log noise"